quaternaire.csl

```diff
@@ -231,7 +231,7 @@
     </layout>
   </citation>
   <!-- ***************** Bibliographie ***************** -->
-  <bibliography entry-spacing="1" line-spacing="1">
+  <bibliography>
```

Comment:
This is fine, but any reason for the change? These are the default values, so this change wouldn't actually change the citation style output.

Reply:
yes, in ms word and LO writer, it creates two blank lines, and if i suppress it, i have no blank lines any more. It's strange, if it is the default values...

![image](https://user-images.githubusercontent.com/61885411/104159755-2e5c7600-53f0-11eb-802d-39b3784dabcc.png)